install-instructions.php

```diff
@@ -359,7 +359,16 @@
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_LINUX; ?>" class="btn btn-lg btn-default"><i class="icon-linux"></i> Linux</a>
 		      <a href="<?php echo $DOWNLOAD_CLIENT_DESKTOP_TEST_SOURCES; ?>" class="btn btn-lg btn-default"><i class="icon-archive"></i> Sources</a>
 		    </div>
-		    <?php } ?>
+		    <?php } else {?>
+			    <h4>Desktop Clients <small><?php echo $VERSIONS_CLIENT_DESKTOP_TESTING; ?></small></h4>
+	 		    <p>Our desktop team provides regular nightly builds. Please report any issues to the <a href="https://github.com/owncloud/mirall/issues" target="_blank">tracker</a>.</p>
+			    <div class="btn-group">
```

Comment:
That should say href="https://github.com/owncloud/client/issues" 
instead of mirall. right?